README.md

```diff
@@ -75,7 +87,21 @@ The use of `--recurse-submodule` is necessary if the user wants the examples dat
 git submodule update --init
 ```
 
-### Build with CMake
+To create a conda environment with all the dependencies for building CIL run:
+
+```bash
+sh scripts/create_local_env_for_cil_development.sh -n NUMPY_VERSION -p PYTHON_VERSION -e ENVIRONMENT_NAME
+```
+
+Or with the CIL build and test dependencies:
+```bash
+sh scripts/create_local_env_for_cil_development_tests.sh -n NUMPY_VERSION -p PYTHON_VERSION -e ENVIRONMENT_NAME
+```
```

Comment:
I wonder if we could add another option to the script, like `-t` to handle test dependencies

Reply:
probably, I added the script I've been using, you can also specify and install a cil version... but that didn't seem necessary information for the readme